src/app.service.ts

```
@@ -1,8 +1,289 @@
 import { Injectable } from '@nestjs/common';
+import axios from 'axios';
+import * as fs from 'fs';
+import * as path from 'path';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)